otx/mpa/seg/__init__.py

```diff
@@ -17,3 +17,4 @@
 # flake8: noqa
 from . import evaluator, exporter, inferrer, stage, trainer
 from .semisl import inferrer, trainer
+from .incremental import inferrer, trainer
```

Comment:
Is this okay? What happens someone call from otx.mpa.seg import trainer?

Reply:
I missed that point. Thanks. It has to call inferrer and trainer inside incremental, but it wasn't. I will fix.

Reply:
Actually it would not matter because each trainer and inferrer was using other class name.

Reply:
Do we need these kinds of imports? Maybe we can remove all kinds of import by adding `__init__.py` inside of each folders. And in this point, we can see the `__init__.py` under `incremental` folder, but can't find in `semisl` folder. we need to unify the file arhictecture.

Reply:
semisl has it, but with no content. I want to ask @sungmanc if it's okay to partially update the convention and follow pre-commit test rules, then I will revise my updated code to match with otx format including __init__.

Reply:
This code still seems to have a problem. This is because inferrer and trainer are imported redundantly, so only semisl is imported here. Looks like you need to fix it.

Reply:
Well,, it is not. Because the class name is totally different, but i will revise it like OTX init.

Reply:
> semisl has it, but with no content. I want to ask @sungmanc if it's okay to partially update the convention and follow pre-commit test rules, then I will revise my updated code to match with otx format including `__init__`.

Ah, I missed the MPA architecture for `__init__.py`. I didn't realize that most of the modules in MPA used direct import(i.e. `from .incremental import inferred, trainer`). I agreed with the current implementation. Let's discuss MPA refactoring later.

For the Harim's opinion, I also agree with this idea, how about using the one like as below:
``` from otx.mpa.seg.incremental.inferrer import IncrSegInferrer ```

@harimkang @supersoob

Reply:
Refactoring all kind of `__init__.py` included in MPA could be handled by another PR because we also need to enable `mypy` and `pylint`, `flake8` for MPA.

Reply:
okay:)) I realized I got it wrong. I was obsessed by my thought. Sorry. I will revise it as just sungman said.